### PR TITLE
Travis: run tests on master

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Travis did not test on latest Orange master.

##### Description of changes
Add scripts to test on master.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ajdapretnar** [comment]
@lanzagar @ales-erjavec @PrimozGodec Could any of the Travis experts help me here. I don't know why Travis is unable to find the files. They seem to be in the right place. @markotoplak confirmed this.

File: .travis.yml [hunk 1]
```diff
@@ -39,6 +44,7 @@ before_install:
 install:
     - travis_wait pip install -r requirements.txt
     - pip install sip pyqt5==5.11.*
+    - source $TRAVIS_BUILD_DIR/.travis/install_orange.sh
```
Comment (astaric) [inline .travis.yml]: you use underscore here, but - in the filename

Comment (ajdapretnar) [reply .travis.yml]: We must be blind. 🙈

Comment (lanzagar) [inline .travis.yml]: Orange is (unnecessarily) installed already 2 lines above when installing requirements.txt
This new install_orange script then uninstalls it and installs the selected version (e.g. master)
You could try moving this up, if it still works correctly in that order, and avoid some extra work.

**Pull request merged**